components/eam/src/physics/clubb/advance_windm_edsclrm_module.F90

```diff
@@ -440,6 +456,182 @@ subroutine advance_windm_edsclrm &
 
     endif ! .not. l_predict_upwp_vpwp
 
+    if ( l_perturbed_wind ) then
+
+       !----------------------------------------------------------------
+       ! Prepare tridiagonal system for horizontal winds, um and vm
+       !----------------------------------------------------------------
+
+       ! Momentum surface fluxes, u'w'|_sfc and v'w'|_sfc, are applied through
+       ! an implicit method, such that:
+       !    x'w'|_sfc = - ( u_star(t)^2 / wind_speed(t) ) * xm(t+1).
+       l_imp_sfc_momentum_flux = .true.
+       ! Compute wind speed (use threshold "eps" to prevent divide-by-zero
+       ! error).
+       wind_speed_pert = max( sqrt( (um_pert)**2 + (vm_pert)**2 ), eps )
+       ! Compute u_star_sqd according to the definition of u_star.
+       u_star_sqd_pert = sqrt( upwp_pert(1)**2 + vpwp_pert(1)**2 )
+
+       ! Compute the explicit portion of the um equation.
+       ! Build the right-hand side vector.
+       call windm_edsclrm_rhs( windm_edsclrm_um, dt, nu10_vert_res_dep, & ! In
+                               Km_zm, um_pert, um_tndcy,                & ! In
+                               rho_ds_zm, invrs_rho_ds_zt,              & ! In
+                               l_imp_sfc_momentum_flux, upwp_pert(1),   & ! In
+                               rhs(:,windm_edsclrm_um)                  ) ! Out
+
+       ! Compute the explicit portion of the vm equation.
+       ! Build the right-hand side vector.
+       call windm_edsclrm_rhs( windm_edsclrm_vm, dt, nu10_vert_res_dep, & ! In
+                               Km_zm, vm_pert, vm_tndcy,                & ! In
+                               rho_ds_zm, invrs_rho_ds_zt,              & ! In
+                               l_imp_sfc_momentum_flux, vpwp_pert(1),   & ! In
+                               rhs(:,windm_edsclrm_vm)                  ) ! Out
+
+
+       ! Store momentum flux (explicit component)
+
+       ! The surface flux, x'w'(1) = x'w'|_sfc, is set elsewhere in the model.
+!      upwp(1) = upwp_sfc
+!      vpwp(1) = vpwp_sfc
```

Comment:
I hate to see abandoned code be commited in a PR, do we need to keep these lines around?

Reply:
Took them out!